opm/input/eclipse/Schedule/Schedule.hpp

```diff
@@ -30,6 +30,7 @@
 #include <unordered_map>
 #include <utility>
 #include <vector>
+#include <functional>
```

Comment:
Very good.  Would you mind keeping the headers in sorted order, please?

Reply:
Done. Looking at the current sorting the criteria was not 100% clear and I might have changed it.